ui/src/main/java/com/midtrans/sdk/ui/models/SavedCreditCard.java

```diff
@@ -0,0 +1,29 @@
+package com.midtrans.sdk.ui.models;
+
+import com.midtrans.sdk.core.models.snap.SavedToken;
+import com.midtrans.sdk.core.models.snap.promo.PromoResponse;
+
+import java.io.Serializable;
+import java.util.List;
+
+/**
+ * Created by ziahaqi on 3/6/17.
+ */
+
+public class SavedCreditCard implements Serializable{
+    private List<SavedToken> savedCards;
```

Comment:
Let's discuss about data models here.

In new `core` module I don't use getter and setter. Instead I use `final`.

So it wil be like 

```java
public final List<SavedToken> savedCards;
public final List<SavedToken> promos;
```